DEVELOPMENT.md

```diff
@@ -1,330 +1,96 @@
-functionality MUST have a corresponding issue and discussion, and will require
-consensus from the editors in order to be introduced or to have any related Pull
-Requests accepted and merged.
-
-A one week (7 day) period prior to merge will be provided to allow sufficient
-review time of pull requests related to data schemas or project functionality.
-Exceptions may be made for essential documentation, or to allow for immediate
-"hotfix" of security issues or functionality breaking items.
-
-## Ontology Structure
-
-This repository hosts [JSON Schema](https://json-schema.org/) which it uses
-to create [JSON-LD](https://json-ld.org/).
-
-All JSON Schema must have an `$id` property, which must resolve to the
-JSON-Schema Document.
-
-For example, see
-[https://w3id.org/traceability/schemas/Person.json](https://w3id.org/traceability/schemas/Person.json).
-
-We are currently 🚧 EXPERIMENTING 🚧 with injecting JSON-LD concepts like `@id`
-and `@type` in JSON Schema using `$comment`.
-
-For example see:
-
-```json
-"$id": "https://w3id.org/traceability/schemas/Person.json",
-"$schema": "http://json-schema.org/draft-07/schema#",
-"$comment": "{\"term\": \"Person\", \"@id\": \"https://schema.org/Person\"}",
-"title": "Person",
-"description": "A person",
 ```
-
-These attributes are then used to deterministically build a JSON-LD context
-hosted at:
-
-[https://w3id.org/traceability/v1](https://w3id.org/traceability/v1)
-
-This context can then be used to produce verifiable credentials for supply chain
-traceability, for example:
-
-```json
-{
-  "@context": [
-    "https://www.w3.org/2018/credentials/v1",
-    "https://w3id.org/traceability/v1"
-  ],
-  "id": "https://example.com/123",
-  "type": ["VerifiableCredential"],
-  "issuer": {
-    "id": "did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM"
-  },
-  "issuanceDate": "2020-03-10T04:24:12.164Z",
-  "expirationDate": "2029-03-10T04:24:12.164Z",
-  "credentialSubject": {
-    "id": "did:example:123",
-    "type": "Place",
-    "address": {
-      "type": "PostalAddress",
-      "organizationName": "Bednar - Kutch",
-      "streetAddress": "8440 Khalid Canyon",
-      "addressLocality": "Gislasonland",
-      "addressRegion": "Ohio",
-      "postalCode": "96546",
-      "addressCountry": "Liechtenstein"
-    }
-  },
-  "proof": {
-    "type": "Ed25519Signature2018",
-    "created": "2021-02-09T21:54:29.223Z",
-    "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..1Xba7-B-YTstwnbIewQFpbMgGfOdRuoDLEBezZlb4M1qMwI9GCUAbeTCsGCPd62XLwyNjPqb9aJGj_xk7iUiBw",
-    "proofPurpose": "assertionMethod",
-    "verificationMethod": "did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM#z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM"
-  }
-}
+git clone https://github.com/w3c-ccg/traceability-vocab.git
+cd traceability-vocab
+npm i
 ```
 
-Inside JSON Schema, you will see things like —
+You can local a version of the [spec](https://w3c-ccg.github.io/traceability-vocab/) you can run:
```

Comment:
What does "You can local a version of" mean?

Reply:
I think i meant to say, "To run a local version"

Reply:
or "You can run a local version"

Reply:
Updated.